charts/opentelemetry-collector/README.md

```diff
@@ -107,6 +108,73 @@ agentCollector:
     mountPropagation: HostToContainer
 ```
 
+### Configuration for Kubernetes container logs
+
+The collector can be used to collect logs sent to standard output by Kubernetes containers.
+This feature is disabled by default. It has the following requirements:
+
+- It needs agent collector to be deployed, which means it will not work if only standalone collector is enabled.
+- It requires the [contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) version
+of the collector image.
+
+To enable this feature, set the  `agentCollector.containerLogs.enabled` property to `true` and replace the collector image.
+Here is an example `values.yaml`:
+
+```yaml
+agentCollector:
+  containerLogs:
+    enabled: true
+
+image:
+  repository: otel/opentelemetry-collector-contrib
+
+command:
+  name: otelcontribcol
+```
+
+The way this feature works is it adds a `filelog` receiver on the `logs` pipeline. This receiver is preconfigured
+to read the files where Kubernetes container runtime writes all containers' console output to.
+
+#### :warning: Warning: Risk of looping the exported logs back into the receiver, causing "log explosion"
+
+The container logs pipeline uses the `logging` console exporter by default.
+Paired with the default `filelog` receiver that receives all containers' console output,
+it is easy to accidentally feed the exported logs back into the receiver.
+
+To prevent this, the default configuration of the receiver excludes logs from the collector's containers.
+
+If you want to include the collector's logs, make sure to replace the `logging` exporter
+with an exporter that does not send logs to collector's standard output.
```

Comment:
Please also add warning that `--log-level=debug` command line option should be avoided when collecting Collector's own logs since at debug level the Collector may produce logs per each collected input log item, which again can result in amplified loop.

Reply:
Added note in https://github.com/astencel-sumo/opentelemetry-helm-charts/commit/8a123a1ec05bf4197fa2f8edc89bba4dd1b810e2.